{"labels": [{"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "TestLoginP"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "host", "value": "G50-75M"}, {"name": "thread", "value": "15668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login_pytest"}], "historyId": "2ccd9b197d81af3dca9b411f49db3e45", "fullName": "TestCases.test_login_pytest.TestLoginP#test_login_0_failed_by_wrong_datas[data2]", "uuid": "de1b4e25-a200-464d-8325-e60cffbb7902", "start": 1561889724552, "name": "test_login_0_failed_by_wrong_datas[data2]", "status": "passed", "parameters": [{"name": "data", "value": "{'user': '186847205531', 'password': 'python', 'check': '请输入正确的手机号'}"}], "stop": 1561889725180}